plugins/content_distribution/lib/api/KalturaCaptionDistributionInfo.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @package plugins.contentDistribution
+ * @subpackage api.objects
+ *
+ */
+class KalturaCaptionDistributionInfo extends KalturaObject{
```

Comment:
Instead of repeating fields from caption asset, use the asset id or extend the KalturaCaptionAsset.